docs/kratos/emails-sms/15_account-activation-email-verification.mdx

```diff
@@ -3,12 +3,8 @@ id: account-activation-email-verification
 title: Setting up account activation and email verification
```

Comment:
Author's comment - I haven't really done anything to this doc aside from moving it to this category. I KINDA feel this is the place for this content but I'm not 100% sure. 

If the placement is okay then we can work on it in the near future.

Reply:
I think this should be moved to self-hosted. All of this is set up and enabled out of the box on Ory Cloud and you only might have a custom identity schema that does not have verification - cant think of another reason why this would be relevant in Cloud.

Reply:
Nice, will do :)

Reply:
moved to self-hosted